library/src/main/java/io/github/douglasjunior/androidSimpleTooltip/SimpleTooltip.java

```diff
@@ -599,6 +603,8 @@ public SimpleTooltip build() throws IllegalArgumentException {
                 SimpleTooltipUtils.setTextAppearance(tv, mDefaultTextAppearanceRes);
                 tv.setBackgroundColor(backgroundColor);
                 tv.setTextColor(textColor);
+                tv.setTypeface(Typeface.createFromAsset(context.getAssets(), fontPath));
```

Comment:
In both cases, it should be checked whether `fontPath` or `fontSize` have been informed before using.

Example:

```jsvs
if (fontPath != null) {
    tv.setTypeface(Typeface.createFromAsset(context.getAssets(), fontPath));
}
```